modules/oadp-timeouts.adoc

```diff
@@ -0,0 +1,226 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: REFERENCE
+[id="oadp-timeouts_{context}"]
+= OADP timeouts
+
+Extending a timeout allows complex or resource-intensive processes to complete successfully without premature termination. This configuration can reduce the likelihood of errors, retries, or failures.
+
+Ensure that you balance timeout extensions in a logical manner so that you do not configure excessively long timeouts that might hide underlying issues in the process. Carefully consider and monitor an appropriate timeout value that meets the needs of the process and the overall system performance.
+
+The following are various OADP timeouts, with instructions of how and when to implement these parameters:
```

Comment:
Usually we wouldn't lead into a section heading with a `:`, but also maybe this line should be at the assembly-level (between the `include`s for this `oadp-timeouts.adoc` and the next module), since theoretically this module _could_ get used in another assembly and might not have the exact modules in the same `include` order.

Reply:
If possible I'd like to leave this the way it is.